guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -707,7 +707,7 @@ class SampleEventObserverThatModifiesInputs
 
 15.1. Use prepared statements for SQL queries.
 
-15.2. Broken Authentication protection.
+###15.2. Broken Authentication protection.
```

Comment:
@kalpmehta, you need to remove all of the hashtags that you added. They're rendering as plaintext in the HTML.